pkg/config/schema.go

```diff
@@ -36,12 +36,44 @@ type Logs struct {
 	Colors  bool `yaml:"colors" json:"colors" mapstructure:"colors"`
 }
 
+type Command struct {
+	Name        string            `yaml:"name" json:"name" mapstructure:"name"`
+	Description string            `yaml:"description" json:"description" mapstructure:"description"`
+	Env         []CommandEnv      `yaml:"env" json:"env" mapstructure:"env"`
+	Arguments   []CommandArgument `yaml:"arguments" json:"arguments" mapstructure:"arguments"`
+	Flags       []CommandFlag     `yaml:"flags" json:"flags" mapstructure:"flags"`
+	Steps       []string          `yaml:"steps" json:"steps" mapstructure:"steps"`
+	Commands    []Command         `yaml:"commands" json:"commands" mapstructure:"commands"`
```

Comment:
nit: how come the struct on the left is plural, singular in the middle, and plural on the right ? Shouldn't we be consistent across the board and rename `CommandArgument` to `CommandArguments` ?

Reply:
not sure what you mean @nitrocode.

in this statement
```
Commands    []Command         `yaml:"commands" json:"commands" mapstructure:"commands"`
```

`Commands` is what we be used in the code, and this is a list of `Command` structure, which describes a single command. `[]Command` is the type.

`commands` in this `yaml:"commands" json:"commands" mapstructure:"commands"`  is how it defined in YAML:

```
commands:
   - ....
```